src/cljs/main/broadfcui/page/library/research_purpose.cljs

```diff
@@ -3,67 +3,131 @@
    [dmohs.react :as react]
    [clojure.string :as string]
    [broadfcui.common.filter :as filter]
+   [broadfcui.common.flex-utils :as flex]
+   [broadfcui.common.icons :as icons]
    [broadfcui.common.links :as links]
    [broadfcui.common.style :as style]
+   [broadfcui.components.autosuggest :refer [Autosuggest]]
    [broadfcui.components.checkbox :refer [Checkbox]]
    [broadfcui.components.modals :as modals]
    [broadfcui.utils :as utils]
    ))
 
 
+(defn- render-doid [doid]
+  (-> doid
+      (string/split "/")
+      last
+      (string/replace-all #"_" " ")))
```

Comment:
I suggest rendering this as colon-delimited instead of space ("DOID:123" instead of "DOID 123"). That's how they do it at http://disease-ontology.org/ and that seems a good standard.